style/modules/scrollCarousel.css

```diff
@@ -0,0 +1,13 @@
+.scrollCarousel {
+    max-height: 120px;
+    border: 3px solid #dddddd;
+    display: flex;
+    overflow-x: auto
+}
+
+.scrollCarousel .item{
```

Comment:
couple smaller things:
- you should have a space between `item` and the left brace
- subcomponent names start with the module name, for example `.scrollCarousel--item`